pkg/cortex/modules.go

```diff
@@ -592,7 +673,13 @@ var modules = map[moduleName]module{
 		stop: (*Cortex).stopCompactor,
 	},
 
+	SingleTableManager: {
```

Comment:
I don't think a separate module is needed in this case. The single binary should use `TableManager`.